verification/verify_test.go

```diff
@@ -217,6 +217,12 @@ func TestVerify(t *testing.T) {
 	assert.Equal(ErrPixelsAbsent, err)
 	assert.Equal("mno", string(res.ManifestID)) // Still return best result
 
+	// A high score should still fail if pixel checking fails
+	verifier.results = &Results{Score: 21.0, Pixels: []int64{-1, -2}}
+	res, err = sv.Verify(&Params{ManifestID: "vws", Results: data})
+	assert.Equal(ErrAudioMismatch, err)
```

Comment:
This test actually fails locally because we need to set the stub error to `ErrAudioMismatch` as opposed to setting up the test to fail pixel checking.

CI passes because [test.sh](https://github.com/livepeer/go-livepeer/blob/bbb72c0f54a9f70ed65611b74abcc52add85bead/test.sh#L61) is not running tests for the `verification` package...Let's update `test.sh` to also run tests in the `verification` package.

Reply:
it should run `go test ./...` - i'll do it in a separate PR

Reply:
> CI passes because test.sh is not running tests for the verification package

🤦‍♂

Reply:
https://github.com/livepeer/go-livepeer/pull/1396

Reply:
https://github.com/livepeer/go-livepeer/pull/1396 - test.sh should run all tests

Let's merge that ^ first, then I'll pull it into this branch

Reply:
Done, and I removed @j0sh 's test because it no longer applies here.

Reply:
After resolving this comment - https://github.com/livepeer/go-livepeer/pull/1384#pullrequestreview-362897982 - I was able to restore j0sh's test (changed to check audio instead) and it passes as expected.